native_client/deepspeech.h

```diff
@@ -4,83 +4,118 @@
 
 #include <cstddef>
 
-typedef struct _DeepSpeechPrivate DeepSpeechPrivate;
+namespace DeepSpeech
+{
 
-class DeepSpeech {
-  private:
-    DeepSpeechPrivate* mPriv;
+  typedef struct _Private Private;
```

Comment:
Because doing the former would require including the Tensorflow headers in the deepspeech header, and those headers aren't publicly available.